packages/data-context/src/sources/migration/shouldShowSteps.ts

```diff
@@ -30,7 +30,7 @@ export function isDefaultTestFiles (config: OldCypressConfig, type: 'component'
     ? getComponentTestFilesGlobs(config)
     : getIntegrationTestFilesGlobs(config)
 
-  return testFiles.length === 1 && testFiles[0] === '**/*'
+  return testFiles.length === 1 && testFiles[0] === '**/*.{js,ts,jsx,tsx,coffee}'
```

Comment:
I'm so confused how this is working at all! I thought that `isDefaultTestFiles` would only be true if the testFiles was `**/*`, which I believe is the 9.x default. I'm quite surprised this didn't break everything. I thought this was used to determine if the user, when migration, has a custom `testFiles` or not.

Maybe this is just working because you changed https://github.com/cypress-io/cypress/pull/20454/files#diff-c2c1f114b54072e7f036c272e41454586ddba1378ada7bf61d38049e05e3d8f5R17 to match?

Should this be a constant perhaps, like `defaultMigrationSpecPattern = '**/*.{js,ts,jsx,tsx,coffee}'`?